packages/expo-dev-menu/app/hooks/useDevSettings.tsx

```diff
@@ -88,8 +88,7 @@ export function useDevSettings() {
   }, []);
 
   const openRNDevMenu = React.useCallback(async () => {
-    await DevMenu.openDevMenuFromReactNative();
-    DevMenu.closeMenu();
```

Comment:
Not necessary anymore, we have to close this from the native side to ensure we first remove the expo-dev-menu overlay and then show the DevMenu from React Native otherwise both menus get closed. 

E.g.

https://github.com/expo/expo/assets/11707729/f7cea942-0b28-4716-89db-d4bfc3ceed43


That's the reason for the changes on packages/expo-dev-menu/ios/Modules/DevMenuInternalModule.swift